geometry_msgs/Pose[] poses
---
bool result
